test/perf/kernel/stockcorr.jl

```diff
@@ -4,7 +4,6 @@
 
 #STOCKCORR - The original, unoptimised code that simulates two correlated assets
 function stockcorr()
-
     ## Correlated asset information
```

Comment:
the extra line of separation here is completely redundant. the indentation already serves this exact purpose